native/src/widget.rs

```diff
@@ -231,4 +239,14 @@ where
     ) -> Option<overlay::Element<'a, Message, Renderer>> {
         None
     }
+
+    /// Name of the widget if there is any
+    fn name(&self) -> String {
+        String::new()
+    }
+
+    /// Returns the id of the widget
+    fn id(&self) -> Option<Id> {
+        None
+    }
```

Comment:
Doc comments could give some context about what the `name` and `id` are used for.

Not necessarily important now, but we'll want more documentation when it's upstreamed to Iced.

Reply:
Ya